### ISSUE TITLE
Window docking shadow is underneath other application windows

### ISSUE THREAD

**Issue by chriswilty** [ScottLogic/StockFlux#993]
When dragging a window to dock / snap it to another, the shadow is underneath other application windows, even if the window you are docking to is the topmost window on the screen. Is this something we can fix via code / config, or an OpenFin issue, or an OS quirk that we have no control over?

In the image below, the full-sized window is being dragged to dock against the smaller window. There is a Notepad++ window open beneath. The snap shadow that shows where the dragged window will dock is _beneath_ the Notepad++ window.

![window_docking_shadow](https://user-images.githubusercontent.com/15246391/49384224-75a4b280-f712-11e8-8584-29b072a729e9.png)

**Comment by AndySmithSL** [comment]
I've seen another variation of this, not sure it should be raised as a separate issue.

When dragging a 'full view' window to the edge of the screen in order to maximise it, the screen is displayed as half opaque / transparent and other windows are visible behind it.

**Comment by chriswilty** [comment]
On my pc, dragging _any_ stock flux window makes it semi-transparent. Is that an OS thing, or configurable for an openfin application? In any case, it's probably a separate issue.

**Comment by AndySmithSL** [comment]
Yep, dragging a stock flux window does make it transparent but when you drop it again it returns to normal. However with dragging it to the edge of the screen to maximise it, the window remains transparent. 
I did just notice that this only happens when you drag it to the top of the screen. Seems fine when you drag it to the left and right edges. Drag it to the bottom you just lose it altogether.

**Comment by chriswilty** [comment]
Corresponding issue in Layouts Service repo:
https://github.com/HadoukenIO/layouts-service/issues/205
